app/src/state/ducks/expenditures.js

```diff
@@ -131,7 +132,15 @@ export function getExpenditures(expenditureSearchAttrs) {
       const response = await api.getExpenditures(expenditureSearchAttrs);
       if (response.status === 200) {
         const data = normalize(await response.json(), schema.expenditure);
-        dispatch(addEntities(data.entities));
+       
```

Comment:
Compensating for an extraneous undefined nested property returned from  
const data = normalize(await response.json(), schema.expenditure);

Returned: expenditures: {undefined: { [list]